docs/source/discord/resources/channel/methods/get_reactions.rst

```diff
@@ -9,14 +9,21 @@ Example
 
 .. code:: c
 
-   struct discord_user **ret;
+   void done_get_reactions(struct discord *client, void *data, const struct discord_users *ret)
+   {
+     if (ret)
```

Comment:
Please check the [guide](https://gist.github.com/lcsmuller/a5f2b205c3871888656b86825db90187#example-3) for the new method of iterating our built-in array types.